doc/api.md

```diff
@@ -537,6 +538,33 @@ which is `details.internal_name` and falls back to `title`.
 - `document_type` *(required)*
   - The `document_type` value that returned editions has.
 
+## `GET /v2/editions`
+
+Retrieves a paginated list of editions for the provided query string
+parameters.
+
+### Query string parameters
+
+- `fields[]` *(optional)*
+  - Accepts an array of: "analytics_identifier", "base_path",
+    "content_id", "description", "document_type", "locale",
+    "public_updated_at", "schema_name", "title"
+  - Determines which fields will be returned in the response, if omitted all
+    fields will be returned.
```

Comment:
I think we probably need details as an option? the docs for get content seem inconsistent with that actually returns (which is, of course, everything ever)